src/main/java/racingcar/model/TryCount.java

```diff
@@ -0,0 +1,28 @@
+package racingcar.model;
+
+public class TryCount {
+
+    private static final String TRY_COUNT_LESS_THAN_ZERO = "[ERROR] 시도 횟수는 1회 이상이어야 합니다.";
+    private static final int MINIMUM_TRY_COUNT = 1;
+
+    private int tryCount;
+
+    public TryCount(int tryCountInput) {
+        validateTryCount(tryCountInput);
+        this.tryCount = tryCountInput;
+    }
```

Comment:
생성자 파라미터 네이밍에 `input`이 들어갈 이유가 있나요? 🤔 
```suggestion
    public TryCount(int tryCountInput) {
        validateTryCount(tryCountInput);
        this.tryCount = tryCountInput;
    }
```